### DISCUSSION TITLE
Remove the vanilla flavour

### DISCUSSION THREAD

**Discussion by marrus-sh** [cat-KIBI-glitch/mastodon#22] [Request for Comments]
I love the taste of vanilla, however…

There are two reasons why GlitchSoc maintains/maintained the vanilla flavour alongside the glitch one:

1. Sometimes new features would be added to upstream and not ported immediately. The vanilla flavour allows access to those features until porting is successful.

2. Avoiding merge conflicts.

Regarding (1), the pace of upstream development has slowed, and I think we should support all features in `glitch` that we are going to support at all.

Regarding (2), a deleted/modified merge conflict is trivial to resolve.

So I’m not sure it is worth keeping around. (Note: Additional flavours significantly increase the time and memory usage of asset compilation.)

Any thoughts on this?

**Comment by aescling** [comment]
i’m very fine with this. personally i’ve never used the vanilla flavour anyway

on a similar note, is there a good reason to not remove the low-contast flavour?

**Reply by marrus-sh** [reply]
uh, skins are fairly inexpensive i think, so i’m not sure there’s a reason to get rid of any which are actively maintained
